developer/releases/4.1/release-notes.md

```diff
@@ -1,45 +1,85 @@
 ---
 layout: doc
-title: Qubes R4.1 release notes
+title: Qubes OS 4.1 release notes
 permalink: /doc/releases/4.1/release-notes/
 ---
 
-Qubes R4.1 release notes
-========================
+## New features and improvements since Qubes 4.0
 
-New features since 4.0
-----------------------
+- Optional qubes-remote-support package now available from repositories
+  (strictly opt-in, no package installed by default; no new ports or network
+  connections open by default; requires explicit connection initiation by the
+  user, then requires sharing a code word with the remote party before a
+  connection can be established; see
+  [#6364](https://github.com/QubesOS/qubes-issues/issues/6364) for more
+  information)
+- Qubes firewall reworked to be more defensive (see
+  [#5540](https://github.com/QubesOS/qubes-issues/issues/5540) for details)
+- Xen upgraded to version 4.12
+- Dom0 operating system upgraded to Fedora 31
+- Default desktop environment upgraded to Xfce 4.14
+- Upgraded default template releases
+- Experimental support for GUI running outside of dom0 (hybrid mode GUI domain
+  without real GPU passthrough; see
+  [#5662](https://github.com/QubesOS/qubes-issues/issues/5662) for details)
+- Support for audio server running outside of dom0 ("Audio domain")
+- sys-firewall and sys-usb are now disposables by default
+- Grub in UEFI mode
```

Comment:
This makes it sound like GRUB has been switched from legacy mode into UEFI mode. I suggest something like:

> GRUB is used to boot on UEFI systems

Reply:
TBH, I had no idea what this line meant, so I just left the language as I found it.

Reply:
> GRUB is used to boot on UEFI systems

What was it before?

Reply:
No, it wasn't.

Reply:
@andrewdavidwong `xen.efi` is loaded directly by the device firmware in R4.0. I assume this line means that the firmware will load `grub.efi` which will then load Xen, but maybe I'm misunderstanding.

Reply:
>> What was it before?
>>
> No, it wasn't.

I think you misread my question. I was just asking what the state of things was before this change (in other words, what changed).

> > @andrewdavidwong `xen.efi` is loaded directly by the device firmware in R4.0. I assume this line means that the firmware will load `grub.efi` which will then load Xen, but maybe I'm misunderstanding.

@marmarek, can you confirm that this is accurate?

Reply:
Exactly, R4.0 (in UEFI boot) has `xen.efi` loaded directly (without any intermediate bootloader) and R4.1 loads grub, which then loads xen. This makes the boot path similar to the legacy boot, and allows the user to modify boot parameters, or choose alternative entry.

Reply:
Sounds good. Thanks for the clarification and confirmation. Pushed an update for this entry.